bugbug/utils.py

```diff
@@ -468,3 +470,20 @@ def should_skip(changeset):
 
 def get_physical_cpu_count() -> int:
     return psutil.cpu_count(logical=False)
+
+
+def extract_metadata(body) -> dict:
```

Comment:
Could you add a short comment in this function and `extract_private` to explain what they do exactly?

Nit: please also add type information to the arguments of the two functions